src/tools/sfdk/sdkmanager.cpp

```diff
@@ -1332,7 +1331,8 @@ bool SdkManager::mapEnginePaths(QString *program, QStringList *arguments, QStrin
 
     const QString cleanSharedHome = this->cleanSharedHome();
     const QString cleanSharedSrc = this->cleanSharedSrc();
-    const QString cleanSharedTarget = this->cleanSharedTarget();
+    QString errorString;
```

Comment:
maybe move that closer to the beginning so that it is available in case we need that for preceding calls in future

Reply:
I prefer declaring variables close to where they are used. But then again, having the cleanShared.*() calls grouped together is also preferable. I moved the declaration now so that it's just before cleanSharedHome() call - it's still relatively close but the calls are neatly together.